rails/rails_trunk/rails_benchmarks/Dockerfile

```diff
@@ -2,7 +2,12 @@ FROM rubybench/ruby:0.5
 MAINTAINER Alan Guo Xiang Tan "https://twitter.com/tgx_world"
 
 RUN apt-get update
-RUN apt-get install -y libncurses5-dev libmysqlclient-dev sqlite3 libsqlite3-dev postgresql mysql-server-5.6
+
+# Install MySql Server in a Non-Interactive mode. Root password will be left blank.
+RUN echo "mysql-server-5.6 mysql-server/root_password password root ''" | debconf-set-selections
+RUN echo "mysql-server-5.6 mysql-server/root_password_again password root ''" | debconf-set-selections
+
+RUN apt-get install -y libncurses5-dev libmysqlclient-dev sqlite3 libsqlite3-dev postgresql mysql-server-5.7
```

Comment:
Though I am not sure if mysql-server is needed inside this image, since we are linking this one to dedicated mysql container.